app/controllers/gardens_controller.rb

```diff
@@ -53,4 +56,14 @@ def garden_params
       plants_attributes: [:name, :times_per_week])
   end
 
+  def next_seven_days
```

Comment:
Now that I think about it some more - seeing as we are using this in a couple places, maybe there is another spot we could move this method that it would make more sense. ApplicationController doesn't really make sense, since it's not really logic related to the controllers. Maybe ApplicationHelper? Or a custom module that could be included in the Garden and Dashboard facades, something like `Schedule`? Not sure if there might be more methods from schedule stuff @bendelonlee that could be included?